Resources/doc/reference/custom_fragment.rst

```diff
@@ -36,12 +36,23 @@ First you need to create a class that extends ``Sonata\ArticleBundle\FragmentSer
         {
             return 'AcmeDummyBundle:Fragment:fragment_my_awesome.html.twig';
         }
+
+        public function getContent()
+        {
+            return array(
+                'fragment' => $fragment,
+                'fields' => $fragment->getSettings(),
+                'foo' => 'bar',
+            );
+        }
     }
 
 
 Using ``settings`` field with ``keys`` option in ``buildEditForm`` method, you can define all elements that compose your fragment.
 Every key is a field that is displayed on the fragment edit form.
 The ``getTemplate`` method allows you to define which template should be used when rendering this fragment type.
+The ``getContent`` method allows you to override the parent getContent method (that returns the fragment and its settings)
```

Comment:
Please quote the second occurence too

Reply:
Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Reply:
sure done